### PR TITLE
DE2184-Use campers contact id on call to GetCampProductDetails

### PR DESCRIPTION
**Pull request by plachmann**

### CODE REVIEW

File: crossroads.net/app/camps/camps.service.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ class CampsService {
     // eslint-disable-next-line prefer-template
     this.campWaiversResource = $resource(__API_ENDPOINT__ + 'api/camps/:campId/waivers/:contactId', { campId: '@campId', contactId: '@contactId' });
     // eslint-disable-next-line prefer-template
-    this.productSummaryResource = $resource(__API_ENDPOINT__ + 'api/camps/:campId/product', { campId: '@campId' });
+    this.productSummaryResource = $resource(__API_ENDPOINT__ + 'api/camps/:campId/product/:camperId', { campId: '@campId', camperId: '@camperId' });
```
Comment (Ganonside) [inline crossroads.net/app/camps/camps.service.js]: Is camperId different from contactId on purpose. If it's meant to be different then I think that's ok but everywhere else it seems to be contactId.

File: Gateway/MinistryPlatform.Translation/Repositories/FormSubmissionRepository.cs [hunk 2]
```diff
@@ -168,7 +168,7 @@ public string GetFormResponseAnswer(int formId, int contactId, int formFieldId)
             var formResponseAnswerId = GetFormResponseAnswerIdForFormFeildFormResponse(formResponseId,formFieldId);
             var responseAnswer =_ministryPlatformRestRepository.UsingAuthenticationToken(apiToken).Get<MpFormAnswer>(formResponseAnswerId, selectColumns);
 
-            return responseAnswer.Response;
+            return responseAnswer == null ? string.Empty:responseAnswer.Response;
```
Comment (Ganonside) [inline Gateway/MinistryPlatform.Translation/Repositories/FormSubmissionRepository.cs]: I would add spaces around the ':' to improve readability.

File: crossroads.net/app/camps/camps.service.js (hunk 1 above)
Comment (plachmann) [inline crossroads.net/app/camps/camps.service.js]: intentional for clarity that this is the camper and not the logged in user

File: Gateway/MinistryPlatform.Translation/Repositories/FormSubmissionRepository.cs (hunk 2 above)
Comment (plachmann) [inline Gateway/MinistryPlatform.Translation/Repositories/FormSubmissionRepository.cs]: change made

**Pull request merged**